### ISSUE TITLE
Crash possibly connected here

### ISSUE THREAD

**Issue by Sunconure11** [Kneelawk/WiredRedstone#21]
https://pastebin.com/uRi2MJKk

**Comment by Kneelawk** [comment]
Wow! That is a lot of mods.

Does the problem go away after removing Immersive Portals? The problem seems to be caused by that mod failing to mixin into a class this mod uses, breaking the class. Are you using the correct version of Immersive Portals?

**Comment by Sunconure11** [comment]
The crash is unrelated to immersive portals. It is blaming an embedded mod I’ve traced to here

**Comment by Kneelawk** [comment]
LibNetworkStack is indeed failing to load, though I can't determine why. The error log shows a mixin failure in Immersive Portals. I'm testing with Immersive Portals now.

**Comment by Kneelawk** [comment]
Ok, I'm having trouble reproducing this. Any tips would be appreciated :)

**Comment by Kneelawk** [comment]
Though Immersive Portals does seem to be causing other issues with LNS.

**Comment by Sunconure11** [comment]
I am using Quilt, which is of note.

**Comment by Sunconure11** [comment]
Also, after I disabled your mod, I ran into this error:

https://pastebin.com/YWqPPFSq

**Comment by Kneelawk** [comment]
Yup, I'm using Quilt, QSL, and QKL in my testing environment. Still not running into this.

The next error you referenced seems unrelated to this issue from what I can tell.
